com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -62,7 +62,6 @@ public sealed class ShaderResources
             public Shader materialErrorPS;
         }
 
-        [Reload("Runtime/Data/PostProcessData.asset")]
         public PostProcessData postProcessData = null;
```

Comment:
I want to add Obsolete attribute here, but it is a bit tricky. We still want to access it internally, so we could support old code path. It means we can not reference name postProcessData. Otherwise we will get warrnings.

Only solution I could think up is and it is not clean:
- `public PostProcessData postProcessData = null;` =>`[FormerlySerializedAsAttribute("postProcessData ")] public PostProcessData m_PostProcessData = null;`
- add `[Obsolete("")] public PostProcessData postProcessData { get =>m_PostProcessData; set { m_PostProcessData  = value;}}`
- add `internal PostProcessData _postProcessData { get =>m_PostProcessData; set { m_PostProcessData  = value;}}`
And this way we would arhieve all needed things:
- Warn users not to use postProcessData.
- Use _postProcessData internally to support old code path.
- Ensure that does not break serialization.
- Keep correct syntax in public eye at least.

WDYT? TBH I do not know if changing public field to property is considered breaking change or not? I think there is some cases were you could have different results.

Reply:
Pretty scared that changing a field to a property will be considered a breaking change, which is actually okay in a major version bump like this one but not for the backports :)

You can put the obsolete tag on it and then where we use it internally use the `#pragma warning disable 618` and `#pragma warning restore 618`. That's fine as we're using it for old code paths but we do not want users to do that.

Reply:
#pragma warning disable 618 // Obsolete warning 
Maybe is the right thing to do

Reply:
Deprecated it in latest commit and add #pragma warning disable 618. Thanks for suggestion it is much better 👍